src/index.html

```diff
@@ -8,6 +8,49 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="container">
+      <div class="stars stars--0">
+        <div class="stars__item stars__item--inactive"></div>
+        <div class="stars__item stars__item--inactive"></div>
+        <div class="stars__item stars__item--inactive"></div>
+        <div class="stars__item stars__item--inactive"></div>
+        <div class="stars__item stars__item--inactive"></div>
+      </div>
+      <div class="stars stars--1">
+        <div class="stars__item stars__item--active"></div>
+        <div class="stars__item stars__item--inactive"></div>
```

Comment:
Remove redundant classes everywhere

```suggestion
        <div class="stars__item"></div>
        <div class="stars__item"></div>
```